addon/components/bourbon-select-field.js

```diff
@@ -38,10 +32,22 @@ export default Component.extend(SelectMixin, {
   hasValue: computed.notEmpty('value'),
   activeOption: null,
 
-  inputValueObserver: observer('value', function() {
-    if (this.get('value') === null) {
+  resetPrompt() {
+    // value, label and selection all need to be set
+    if (this.get('hasPrompt') && (!this.get('value'))) {
       this.set('label', this.get('prompt'));
+    } else if (typeof this.get('value') === 'string') {
```

Comment:
Should probably cache `this.get('value')` in a variable? So it doesn't have to go through `.get()` every time?

Reply:
ok.  i will update in the next PR.